index.js

```diff
@@ -79,18 +79,14 @@ module.exports = {
     "no-trailing-spaces": 1,
     "no-useless-constructor": 1,
     "one-var": [ 1, { "uninitialized": "always", "initialized": "never" } ],
-
-    /* "object-curly-spacing": [ 2, "never" ], */
-
+    "object-curly-spacing": [ 1, "never" ],
```

Comment:
I dunno, these types of rules feel constricting. I've always been of the camp that developers should be able to do what feels right in context when it comes to minor formatting. I think the fact that you used to like it one way, then changed your tune is a good case in point, that it's better to just leave more things lenient and be able to get a feel for what you like.